### PR TITLE
Release/6.3.0

### PR DESCRIPTION
**Pull request by kopijunkie**
### Added

- Internal: Added unit tests for Demo and App components
- Public: Updated supported documents data to include Peru, Colombia as an issuing country option in Country Selection screen when user selects Residence Permit document type and remove Saudi Arabia option for National Identity Card document type.
- Public: Added `CROSS_DEVICE_START` to Tracked events list
- Public: Country Selection screen can now be suppressed for a non-passport document type when configured with a 3-letter ISO code.

### Changed

- Internal: Upgrade Preact from version `8.5.2` to `10.5.4`.
- Internal: Replace `react-modal-onfido` with version `3.11.2` of `react-modal`.
- Internal: Refactor cross device option logic.

### Fixed

- Public: Fixed Woopra module import errors
- Internal: Include `isCrossDevice` property and value in Document, Face capture payload's `sdk_metadata` object for data tracking

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1246-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1246-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1246-pr-onfido-sdk-ui-onfido.surge.sh

**Review by stefaniacardenas — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -819,7 +840,9 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 - NPM (commonjs2) style of importing the library now works
 
-[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/6.2.0...development
+[next-version]: https://github.com/onfido/onfido-sdk-ui/compare/6.3.0...development
+[6.2.1]: https://github.com/onfido/onfido-sdk-ui/compare/6.2.1...6.3.0
```
Comment (anuragajwani) [inline CHANGELOG.md:844]: shouldn't this be 6.3.0? looks like there are two entries for 6.2.1

File: demo/fiddle/demo.details [hunk 2]
```diff
@@ -3,6 +3,6 @@
  authors:
    - Onfido Team
  resources:
-   - https://assets.onfido.com/web-sdk-releases/6.2.0/onfido.min.js
-   - https://assets.onfido.com/web-sdk-releases/6.2.0/style.css
+   - https://assets.onfido.com/web-sdk-releases/6.3.0-rc.1/onfido.min.js
```
Comment (anuragajwani) [inline demo/fiddle/demo.details:6]: why does this say rc.1? There is no specification of release candidate anywhere else in the documentation. Does this get autoupdated upon release?

Comment (kopijunkie) [reply demo/fiddle/demo.details:6]: Yeap. This will get autoupdated on release when the 6.3.0 tag gets pushed to Github.

File: CHANGELOG.md (hunk 1 above)
Comment (kopijunkie) [reply CHANGELOG.md:844]: great spot. will edit that.

**Review by anuragajwani — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1246-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1246-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1246-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1246-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**